lib/log.js

```diff
@@ -74,9 +74,16 @@ Lug.prototype.activityEvent = function(event, uid, request) {
   if (request.headers['user-agent']) {
     info.userAgent = request.headers['user-agent']
   }
-  if (request.payload && request.payload.service) {
-    info.service = request.payload.service
+  try {
```

Comment:
nice, although probably worth a code comment to explain the weirdness for posterity :-)